### ISSUE TITLE
coverage resets a manually set scalaVersion

### ISSUE THREAD

**Issue by fthomas** [scoverage/sbt-scoverage#146]
With sbt-scoverage 1.3.3 this happens in my project:
```
> ++2.10.6
[info] Setting version to 2.10.6
[info] Reapplying settings...
[info] Set current project to refined (in build file:/home/frank/data/code/refined/)
> refinedJVM/scalaVersion
[info] 2.10.6
> coverage
[info] Set current project to refined (in build file:/home/frank/data/code/refined/)
> refinedJVM/scalaVersion
[info] 2.11.7
```
Calling coverage should not change `scalaVersion`. I first noticed this issue here https://github.com/fthomas/refined/issues/72.

**Comment by rorygraves** [comment]
Ouch thats nasty.
My sbt foo is a bit weak, the only client code executed is this 
```scala
  private def toggleCoverage(status:Boolean): State => State = { state =>
    val extracted = Project.extract(state)
    val newSettings = extracted.structure.allProjectRefs map { proj =>
      coverageEnabled in proj := status
    }
    extracted.append(newSettings, state)
  }
```

Which is setting the coverageEnabled flag for each project to true.  I'm guessing there is a misunderstanding of how/what this code is doing.

**Comment by fommil** [comment]
I understand scala 2.10 support was dropped in scoverage.

**Comment by sksamuel** [comment]
Yes it was (was always patchy due to range position stuff).

**Comment by fthomas** [comment]
This issue is unrelated to the dropped 2.10 support:
```scala
> ++2.11.7
[info] Setting version to 2.11.7
[info] Reapplying settings...
[info] Set current project to refined (in build file:/home/frank/data/code/refined/)
> coreJVM/scalaVersion
[info] 2.11.7
> coverage
[info] Set current project to refined (in build file:/home/frank/data/code/refined/)
> coreJVM/scalaVersion
[info] 2.11.8
```
Just tested this with version 1.3.5.

**Comment by travisbrown** [comment]
This is fixed in 1.4.0, right?

**Comment by gslowikowski** [comment]
Right.

**Comment by gslowikowski** [comment]
Fixed by #186
